datafusion/sqllogictest/test_files/csv_files.slt

```diff
@@ -362,3 +362,15 @@ OPTIONS ('format.terminator' E'\r', 'format.has_header' 'true');
 
 statement ok
 drop table stored_table_with_cr_terminator;
+
+
+# check that datetime_format is case sensitive
```

Comment:
that doesn't work, reported a separate issue https://github.com/apache/datafusion/issues/13633